botocore/docs/bcdoc/style.py

```diff
@@ -220,7 +220,7 @@ def end_a(self):
                 if ':' in last_write:
                     last_write = last_write.replace(':', r'\:')
                 self.doc.push_write(last_write)
-                self.doc.push_write(' <%s>`_' % self.a_href)
+                self.doc.push_write(' <%s>`__' % self.a_href)
```

Comment:
They can, but I'm not sure it's strictly needed. I may have found another issue. It seems rst may not be able to do hyperlinked literals like [`this`](https://example.com/).